sensor/common/collector/service_impl.go

```diff
@@ -0,0 +1,132 @@
+package collector
+
+import (
+	"context"
+
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/generated/internalapi/sensor"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/centralsensor"
+	"github.com/stackrox/rox/pkg/grpc/authz/idcheck"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/sensor/common"
+	"github.com/stackrox/rox/sensor/common/message"
+	"google.golang.org/grpc"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+// CollectorService is the struct that manages the collector configuration
+type serviceImpl struct {
+	sensor.UnimplementedCollectorServiceServer
+
+	collectorC chan *sensor.MsgToCollector
+
+	connectionManager *connectionManager
+}
+
+func (s *serviceImpl) Notify(e common.SensorComponentEvent) {
+}
+
+func (s *serviceImpl) Start() error {
+	s.collectorC = make(chan *sensor.MsgToCollector)
```

Comment:
Thanks for the improvements!
We already have one `make` for the channel in `NewService()`, so this one should not be necessary.
It is not strictly a bug, but a style issue (unless something tries to write to that channel before the component is started).

I suggest to remove this `make` and keep the one in `NewService()`.
```suggestion
```